src/libtmux/session.py

```diff
@@ -253,6 +253,9 @@ def new_window(
         if window_name:
             window_args += ("-n%s" % window_name,)
 
+        if window_name == '':
+            window_args += ('-n', '')
```

Comment:
```suggestion
        if window_name is not None and isinstance(window_name, str):
            window_args += ("-n%s" % window_name,)
```

```suggestion
        if window_name is not None:
            window_args += ("-n%s" % window_name,)
```

Would either of these accomplish the same?

Reply:
I don’t think that would do it because a non-empty string would satisfy that condition.

Reply:
Both `''` and `'normal string'` would, no?

Reply:
It should work

Reply:
Would you take the above example as a compromise? 

I think the reason why the above works in one condition is it eliminates the `("-n%s" % window_name,)`. That's where the `"''"` is being lost.

What do you think?